lumigator/python/mzai/backend/backend/config_templates.py

```diff
@@ -11,7 +11,8 @@
         "max_samples": {max_samples},
         "return_input_data": true,
         "return_predictions": true,
-        "storage_path": "{storage_path}"
+        "storage_path": "{storage_path}",
+        "skip_inference": "{skip_inference}"
```

Comment:
I want to say something against this, but... won't 😆 

Also, now seriously, this might be handy for #435 to compare against, since essentially they need to return the same thing.

Reply:
:D yeah, this was really hacky! It is also why I tried to keep everything into a single commit, we might revert just that if for some reason we want to keep the original evaluator after we have already introduced the light eval